components/Votes/Votes.tsx

```diff
@@ -111,60 +105,40 @@ export function Votes() {
   }
 
   return (
-    <OuterWrapper>
-      {getUserIndependentIsLoading() ? (
-        <LoadingSpinner size={300} variant="black" />
-      ) : (
-        <InnerWrapper>
-          <Title>{determineTitle()}</Title>
-          <VoteTimeline />
-          <VotesTableWrapper>
-            <VotesTable
-              headings={<VotesTableHeadings activityStatus={getActivityStatus()} />}
-              rows={determineVotesToShow().map((vote) => (
-                <VotesTableRow
-                  vote={vote}
-                  phase={phase}
-                  selectedVote={selectedVotes[vote.uniqueKey]}
-                  selectVote={(value) => selectVote(value, vote)}
-                  activityStatus={getActivityStatus()}
-                  moreDetailsAction={() => openVotePanel(vote)}
-                  key={vote.uniqueKey}
-                  isFetching={getUserDependentIsFetching() || isCommittingVotes || isRevealingVotes}
-                />
-              ))}
+    <>
```

Comment:
this logic is the same just moved. these wrappers are now shared between page components.